odl/solvers/util/callback.py

```diff
@@ -445,11 +457,64 @@ def __repr__(self):
         return '{}({})'.format(self.__class__.__name__, inner_str)
 
 
+class CallbackPrintTotalTime(Callback):
```

Comment:
Why do we need a new class for this? 
IMO we should instead just add a `cumulative=False` option to the existing `CallbackPrintTiming` and massage the code a bit to cover both cases.

Reply:
Actually this is a great suggestion. Perhaps that should be done instead.

Reply:
Agree. I will make those changes.